configure.py

```diff
@@ -975,43 +974,44 @@ def find_ninja():
                 'cql3/restrictions/statement_restrictions.cc',
                 'cql3/result_set.cc',
                 'cql3/prepare_context.cc',
-                'db/consistency_level.cc',
-                'db/system_keyspace.cc',
-                'db/virtual_table.cc',
-                'db/virtual_tables.cc',
-                'db/system_distributed_keyspace.cc',
-                'db/size_estimates_virtual_reader.cc',
-                'db/schema_applier.cc',
-                'db/schema_tables.cc',
-                'db/cql_type_parser.cc',
-                'db/legacy_schema_migrator.cc',
+                'db/batchlog_manager.cc',
                 'db/commitlog/commitlog.cc',
-                'db/commitlog/commitlog_replayer.cc',
                 'db/commitlog/commitlog_entry.cc',
+                'db/commitlog/commitlog_replayer.cc',
+                'db/config.cc',
+                'db/consistency_level.cc',
+                'db/cql_type_parser.cc',
                 'db/data_listeners.cc',
+                'db/extensions.cc',
                 'db/functions/function.cc',
+                'db/heat_load_balance.cc',
+                'db/hints/host_filter.cc',
                 'db/hints/internal/hint_endpoint_manager.cc',
                 'db/hints/internal/hint_sender.cc',
                 'db/hints/internal/hint_storage.cc',
                 'db/hints/manager.cc',
                 'db/hints/resource_manager.cc',
-                'db/hints/host_filter.cc',
                 'db/hints/sync_point.cc',
-                'db/config.cc',
-                'db/extensions.cc',
-                'db/heat_load_balance.cc',
                 'db/large_data_handler.cc',
+                'db/legacy_schema_migrator.cc',
                 'db/marshal/type_parser.cc',
-                'db/batchlog_manager.cc',
+                'db/per_partition_rate_limit_options.cc',
+                'db/rate_limiter.cc',
+                'db/row_cache.cc',
+                'db/schema_applier.cc',
+                'db/schema_tables.cc',
+                'db/size_estimates_virtual_reader.cc',
+                'db/snapshot-ctl.cc',
+                'db/snapshot/backup_task.cc',
+                'db/sstables-format-selector.cc',
+                'db/system_distributed_keyspace.cc',
+                'db/system_keyspace.cc',
                 'db/tags/utils.cc',
+                'db/view/row_locking.cc',
                 'db/view/view.cc',
                 'db/view/view_update_generator.cc',
-                'db/view/row_locking.cc',
-                'db/sstables-format-selector.cc',
-                'db/snapshot-ctl.cc',
-                'db/rate_limiter.cc',
-                'db/per_partition_rate_limit_options.cc',
-                'db/snapshot/backup_task.cc',
+                'db/virtual_table.cc',
+                'db/virtual_tables.cc',
```

Comment:
Looks like this list was sorted in addition to relocating `row_cache.cc`. I fully approve of sorting (I think all lists should be sorted), but next time please do this in a separate commit, so the meat of this change (relocating `row_cache.cc`) is more clear.

Reply:
Yes, My bad.
I planned to make it in two commits, even maybe initially did :thinking: ?
But then I squashed all commits together after fixing your previous comment :man_shrugging:.
I'll be more patient next time.